tests/kyverno/defaults/generate-default-rolebinding-in-ns-test.yaml

```diff
@@ -0,0 +1,21 @@
+name: test default RoleBinding generation
+policies:
+  - compiled/appuio-cloud/appuio-cloud/10_generate_default_rolebinding_in_ns.yaml
+resources:
+  - generate-default-rolebinding-in-ns/uninit-rolebinding.yaml
+variables: generate-default-rolebinding-in-ns-variables.yaml
+results:
+  - policy: default-rolebinding-in-ns
+    rule: patch-uninitialized-default-rolebinding
+    resource: admin
+    namespace: org-namespace
+    kind: RoleBinding
+    result: skip
+    patchedResource: generate-default-rolebinding-in-ns/uninit-rolebinding-patched.yaml
```

Comment:
I guess the resource is not required here since the policy is skipped?

Reply:
You'd thinks so yes, but no..